test/fast-money-test.js

```diff
@@ -1,22 +1,32 @@
-// import chai from 'chai';
-// // import Round from '../src/Round';
-// import DominationRound from '../src/DominationRound';
-// const expect = chai.expect;
+import chai from 'chai';
+import data from '../data/surveys.js'
+import Game from '../src/Game';
+// import Round from '../src/Round';
+import DominationRound from '../src/DominationRound';
+const expect = chai.expect;
 
-// let dominationRound;
-// beforeEach(() => {
-//   dominationRound = new DominationRound();
-// });
+let dominationRound, game;
+beforeEach(() => {
+  game = new Game(data)
+  dominationRound = new DominationRound(game, 3);
+});
 
-// describe('DominationRound', () => {
+describe('DominationRound', () => {
   
-//   it('should be a function', () => {
-//     expect(DominationRound).to.be.a('function');
-//   });
+  it.only('should be a function', () => {
+    expect(DominationRound).to.be.a('function');
+  });
 
-//   it('should be an instance of DominationRound', () => {
-//     expect(dominationRound).to.be.an.instanceOf(DominationRound);
-//   });
+  it.only('should be an instance of DominationRound', () => {
```

Comment:
Be careful with the `.only` chain, you don't want to forget to run all your tests from time to time